keyboards/hexon38/keymaps/default/keymap.c

```diff
@@ -0,0 +1,426 @@
+#define SLASH KC_SLSH
+#define SPACE KC_SPC
+#define TAB KC_TAB
+#define BKSPC KC_BSPC
+#define ENTER KC_ENT
+#define PERIOD KC_DOT
+
+#define BASE_LAYER KEYMAP
+#define BLANK_LAYER KEYMAP
+#define UNUSED_LAYER KEYMAP(__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__,__)
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    BASE_LAYER(
+//                  ,--------+--------+--------+--------.    ,--------+--------+--------+--------.
+                        W_   ,   E_   ,   R_   ,   T_   ,        Y_   ,   U_   ,   I_   ,   O_   ,
+//|--------+--------+--------+--------+--------+--------|    |--------+--------+--------+--------+--------+--------.
+      Q_   ,  A_CTL ,  S_ALT ,  D_GUI ,  F_SFT ,   G_   ,        H_   ,  J_SFT ,  K_GUI ,  L_ALT ,COLN_CTL,   P_   ,
+//|--------+--------+--------+--------+--------+--------'    `--------+--------+--------+--------+--------+--------|
+      B_   ,   Z_   ,   X_   ,   C_   ,   V_   ,                          M_   ,  COMMA , PERIOD ,  SLASH ,   N_   ,
+//`--------+--------+--------+--------+--------'                      `--------+--------+--------+--------+--------'
+
+//                  ,--------+--------+--------+--------.    ,--------+--------+--------+--------.
+                      LSHIFT ,  SPACE ,   TAB  ,  DEBUG ,       SPACE ,  BKSPC ,  ENTER , RSHIFT
+//                  `--------+--------+--------+--------'    `--------+--------+--------+--------'
+),
+
+    BLANK_LAYER(
+//                  ,--------+--------+--------+--------.    ,--------+--------+--------+--------.
+                      ______ , ______ , ______ , ______ ,      ______ , ______ , ______ , ______ ,
+//|--------+--------+--------+--------+--------+--------|    |--------+--------+--------+--------+--------+--------.
+    ______ , ______ , ______ , ______ , ______ , ______ ,      ______ , ______ , ______ , ______ , ______ , ______ ,
+//|--------+--------+--------+--------+--------+--------'    `--------+--------+--------+--------+--------+--------|
+    ______ , ______ , ______ , ______ , ______ ,                        ______ , ______ , ______ , ______ , ______ ,
+//`--------+--------+--------+--------+--------'                      `--------+--------+--------+--------+--------'
+
+//                  ,--------+--------+--------+--------.    ,--------+--------+--------+--------.
+                      ______ , ______ , ______ , ______ ,      ______ , ______ , ______ , ______
+//                  `--------+--------+--------+--------'    `--------+--------+--------+--------'
+),
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER,
+    UNUSED_LAYER
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`.